decidim-proposals/app/types/decidim/proposals/proposals_type.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    # This type represents a ParticipatoryProcess.
+    ProposalsType = GraphQL::ObjectType.define do
+      interfaces [Decidim::Core::ComponentInterface]
+
+      name "Proposals"
+      description "A proposals component of a participatory space."
+
+      connection :proposals, ProposalType.connection_type do
+        resolve ->(feature, _args, _ctx) {
+                  ProposalsTypeHelper.base_scope(feature).includes(:feature).published
```

Comment:
`published` is already present at the base scope

Reply:
Should we make sure that the feature is also published?

Reply:
No @oriolgual, since the feature's visibility is determined to its own field (participatory process or assembly).